python/pyspark/sql/connect/column.py

```diff
@@ -390,3 +391,61 @@ def __nonzero__(self) -> None:
 
 
 Column.__doc__ = PySparkColumn.__doc__
+
+
+def _test() -> None:
+    import os
+    import sys
+    import doctest
+    from pyspark.sql import SparkSession as PySparkSession
+    from pyspark.testing.connectutils import should_test_connect, connect_requirement_message
+
+    os.chdir(os.environ["SPARK_HOME"])
+
+    if should_test_connect:
+        import pyspark.sql.connect.column
+
+        globs = pyspark.sql.connect.column.__dict__.copy()
+        # Works around to create a regular Spark session
+        sc = SparkContext("local[4]", "sql.connect.column tests", conf=SparkConf())
+        globs["_spark"] = PySparkSession(sc, options={"spark.app.name": "sql.connect.column tests"})
+
+        # Creates a remote Spark session.
+        os.environ["SPARK_REMOTE"] = "sc://localhost"
+        globs["spark"] = PySparkSession.builder.remote("sc://localhost").getOrCreate()
+
+        # TODO(SPARK-41751): Support bitwiseAND
+        del pyspark.sql.connect.column.Column.bitwiseAND.__doc__
+        del pyspark.sql.connect.column.Column.bitwiseOR.__doc__
+        del pyspark.sql.connect.column.Column.bitwiseXOR.__doc__
```

Comment:
quick question, were they failed because of `bitwiseAND` too?

Reply:
nope, Error was Similar Error in all of these
```[UNRESOLVED_ROUTINE] Cannot resolve function `bitwiseAND` on search path [`system`.`builtin`, `system`.`session`, `spark_catalog`.`default`].```

Reply:
Mind creating a JIRA for all? Feel free to either fix the existing JIRA to list all functions, or create a JIRA for each.

Reply:
done updated the JIRA